src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -64,13 +72,23 @@ const defaultFormValues = {
   manualRuneAddress: '',
 }
 
+// The staking Asset on L2, and its matching L1 asset to be used for bridging
+// Ideally, we'd like to use related assets to get the original L1 implementation of the asset but
+// 1. we don't have FOX on Arb as a related asset on any list just yet and most importantly
+// 2. this assumes that the only possible flow is a token on L2 and its original implementation on L1
+// Given the facts that this may not hold true, we may never have this flow generalized like so, and this adds complexity,
+// this is hardcoded for the sake of simplicity
+const assetIds: [AssetId, AssetId] = [foxOnArbitrumOneAssetId, foxAssetId]
```

Comment:
Or alternatively, just pass the original implementation assetId (e.g `l1AssetId`) in and call it a day instead of large comment

Reply:
36ff8666e